tensorflow/python/debug/cli/analyzer_cli_test.py

```diff
@@ -46,6 +46,9 @@
 from tensorflow.python.platform import test
 from tensorflow.python.util import tf_inspect
 
+def matmul_op_name():
```

Comment:
Please add a comment in the code describing why we need this function (e.g., TensorFlow with MKL-DNN integration changes the op name to _MklMatMul).